src/Hint/Annotations.hs

```diff
@@ -18,12 +17,13 @@ getModuleAnnotations :: (Data a, MonadInterpreter m) => a -> String -> m [a]
 getModuleAnnotations _ x = do
     mods <- GHC.mgModSummaries . hsc_mod_graph <$> runGhc GHC.getSession
     let x' = filter ((==) x . GHC.moduleNameString . GHC.moduleName . ms_mod) mods
-    concat <$> traverse (anns . ModuleTarget . ms_mod) x'
+    concatMapM (anns . ModuleTarget . ms_mod) x'
```

Comment:
`concat <$> traverse f xs` is `concatMapM f xs`. I normally use the `concatMapM` from `extra`, but it looks like there's one in `ghc` too!